Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
"Comprobar por" is not an appropriate Spanish phrases verb.
You shouldn't try to make a literal translation from English.
This particular statement might be confusing for native Spanish speakers.

The previous version would translate to English as "... verify that the RDB is up to date", which is the right idea.

Please rephrase or revert this line.